crates/bevy_ecs/src/system/function_system.rs

```diff
@@ -489,3 +506,21 @@ macro_rules! impl_system_function {
 }
 
 all_tuples!(impl_system_function, 0, 16, F);
+
+pub trait AsSystemLabel<Marker> {
```

Comment:
Please add a short doc comment stating this is used for system -> SystemLabel coercion plus a pointer to SystemLabel which illustrates the ergonomics.

It'd also stave off the perpetually missing `#[forbid(missing_docs)]` for the crate.

Reply:
Good call!